component/src/main/ballerina/ftp/Package.md

```diff
@@ -0,0 +1,84 @@
+## Package overview
+The `wso2/ftp` package provides an FTP client and an FTP server listener implementation to facilitate an FTP connection to a remote location. 
+
+### FTP client
+`ftp:Client` connects to an FTP server and performs various operations on the files. It supports `get`, `delete`, `put`, `append`, `mkdir`, `rmdir`, `isDirectory`,  `rename`, `size`, and `list` operations.
+
+An FTP client endpoint is defined using these parameters: `protocol`, `host` and optionally, `port` and `secureSocket`.  Authentication configuration whether is in basicAuth based or private key based or TrustStore,Keystore based can be configure using  secureSocket parameter.
```

Comment:
Change to 'Authentication configuration can be configured using the secureSocket parameter regardless of whether it is basicAuth-based, private key-based, TrustStore-based, or Keystore-based.'